pom.xml

```
@@ -39,13 +39,13 @@
 		<dependency>
 			<groupId>dev.dsf</groupId>
 			<artifactId>dsf-bpe-process-api-v1</artifactId>
-			<version>1.3.1</version>
+			<version>1.5.0</version>
 			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>de.hs-heilbronn.mi</groupId>
 			<artifactId>crypto-utils</artifactId>
-			<version>3.8.0</version>
+			<version>4.0.0</version>
```

DSF process api v1 useses crypto utils 3.8.0. do not upgrade to 4.0.0 unless you include it in a fat process plugin jar.

```suggestion
			<version>3.8.0</version>
```

---

reset with bf0f632